Windows Installer (Silent mode) (#18613)
netdata/netdata: packaging/windows/WINDOWS_INSTALLER.md

-->

# Netdata Installer

For Microsoft Windows, Netdata provides an executable to install and claim Netdata. This binary offers two different
installation modes, which are described in the following sections.

## Graphical User Interface (GUI)

When you double-click the installer, you will be prompted to accept the installation as an administrator since
Netdata will add a service to your system. After this, the installer GUI will guide you through the following steps:

- Welcome: The initial screen provides details about the actions the installer will perform.
- UI License: This screen presents the Netdata Cloud UI. You must accept the license to proceed.
- GPL 3 License: You will then be presented with the GPL 3 License, which governs netdata software.
- Destination: By default, Netdata will be installed in `C:\Program Files\Netdata`. If you prefer a different directory, 
  you can change it here.
- Installation: At this stage, the software will be installed in the selected directory.
- Claim: If you haven’t claimed your agent yet, this screen will allow you to
  [claim](https://github.com/netdata/netdata/blob/master/src/claim/README.md) it. The following fields are available:
 

>     - Token: Your Netdata Cloud token.

Reviewer: ```suggestion
    - Token: Your Netdata Cloud Space's claim token.
```

    - Rooms: The rooms to which you want to assign your node, separated by commas.
    - Proxy: (Optional) The proxy address used to communicate with Netdata Cloud.
    - Insecure connection: By default, Netdata verifies the host certificate. If this option is selected,
      certificate verification will be disabled.
    - Open terminal: Selecting this option will open the `MSYS2 `terminal at the end of the installation.
- Finish: The installation completes.  

## Silent Mode

For faster installation, the silent mode does not display the 

> [cloud license](https://raw.githubusercontent.com/netdata/netdata/master/src/web/gui/v2/LICENSE.md),

Reviewer: ```suggestion
[Netdata Cloud UI License](src/web/gui/v2/LICENSE.md),
```

> or the [GPL3 license](https://www.gnu.org/licenses/gpl-3.0.txt). However, you must use the `/A` option to indicate that

Reviewer: ```suggestion
nor the [GPL3 License](https://www.gnu.org/licenses/gpl-3.0.txt). However, you must use the `/A` option to indicate that
```

> you accept both licenses before proceeding.

Reviewer: ```suggestion
you accept both Licenses before proceeding.
```

In silent mode, the following options are available without additional arguments:

- `/S`: Runs the installer in silent mode.

> - `/A`: Accepts all licenses required by Netdata. Without this option, the installer will not proceed.

Reviewer: ```suggestion
- `/A`: Accepts all Licenses required by Netdata. Without this option, the installer will not proceed.
```

- `/D`: Specifies the destination directory.
- `/T`: Opens the `MSYS2` terminal after installation is complete.
- `/I`: Accepts insecure connections, meaning the hostname will not be verified.

Additionally, the following options require arguments:

> - `/TOKEN=`: Specifies your Netdata Cloud token.

Reviewer: ```suggestion
- `/TOKEN=`: Specifies your Netdata Cloud Space's claim token.
```

> - `/ROOMS=`: Specifies the rooms to which you want to connect the host. Multiple rooms can be specified by separating

Reviewer: ```suggestion
- `/ROOMS=`: Specifies the Room IDs to which you want to connect the node to. Multiple rooms can be specified by separating
```
